qtconsole/completion_widget.py

```diff
@@ -20,9 +21,21 @@ def __init__(self, console_widget):
         super(CompletionWidget, self).__init__()
 
         self._text_edit = text_edit
+        self.setEditTriggers(QtGui.QAbstractItemView.NoEditTriggers)
+        self.setHorizontalScrollBarPolicy(QtCore.Qt.ScrollBarAlwaysOff)
+        self.setSelectionBehavior(QtGui.QAbstractItemView.SelectRows)
+        self.setSelectionMode(QtGui.QAbstractItemView.SingleSelection)
+
+        if os.name == 'nt':
```

Comment:
Let's leave a brief comment here explaining why we need different flags for different systems.

We should also try to get someone with OSX to check that it's working for them.